src/django/api/models/user.py

```diff
@@ -45,8 +44,13 @@ def create_superuser(self, email, password=None, **extra_fields):
         if extra_fields.get("is_superuser") is not True:
             raise ValueError("Superuser must have is_superuser=True.")
 
-        admin_role = Role.objects.get(pk=Roles.ADMINISTRATOR)
-        return self._create_user(email, admin_role, password, **extra_fields)
+        return self._create_user(email, Roles.ADMINISTRATOR, password, **extra_fields)
+
+
+class Roles(models.TextChoices):
+    CONTRIBUTOR = "C"
+    VALIDATOR = "V"
+    ADMINISTRATOR = "A"
```

Comment:
The example in [Django's docs](https://docs.djangoproject.com/en/3.2/ref/models/fields/#enumeration-types) uses nested classes, but I don't have a strong opinion about doing it either way.

Reply:
Or, we could use numeric choices, but then someone reporting on the database needs to consult this repo's source to decode it.